packages/axway-cli-oum/src/org/usage.js

```diff
@@ -51,41 +99,90 @@ export default {
+			console.log(
+				`  ${highlight(`${format(bundle.value)} / ${format(bundle.quota)}`)} ${bundle.units}`
+				+ `  ${renderBar(bundle.percent, 40)}`
+			);
+
+			const table = createTable();
+			for (const [ name, metric ] of Object.entries(bundle.metrics)) {
+				table.push([
+					`  ${bold(metric.name || name)}`,
+					'',
+					{ content: `${highlight(format(metric.value))} ${bundle.units}`, hAlign: 'right' }
+				]);
+
+				const ratio = bundle.ratios[name];
+
+				// render the envs
+				for (let i = 0, len = metric.envs.length; i < len; i++) {
+					const env = metric.envs[i];
+					table.push([
+						`  ${i + 1 === len ? '└─' : '├─'} ${env.name} ${env.production ? gray('Production') : ''}`,
+						`${highlight(format(env.value).padStart(bundleValuePadding))} Transactions${env.tokens && ratio !== 1 ? highlight(` x ${(ratio / 100).toFixed(1)}`) : ''}`,
+						env.tokens ? { content: `${highlight(format(env.tokens))} ${bundle.units}`, hAlign: 'right' } : ''
+					]);
+				}
+			}
+			if (table.length) {
+				console.log();
+				console.log(table.toString());
+			}
 		}
 
-		let pad = 0;
-		const width = 20;
+		// Project usage
 		const table = createTable();
-		const { format } = new Intl.NumberFormat();
-		const metrics = Object.values(usage.SaaS).sort((a, b) => a.name.localeCompare(b.name));
-
-		// loop through once to compute the usage bars
-		for (const info of metrics) {
-			const pct = info.quota ? (info.value / info.quota) : 0;
-			info.color = pct > 85 ? red : pct > 65 ? yellow : green;
-			info.pct = String(Math.round(pct * 100));
-			info.used = Math.ceil(width * pct);
-			pad = Math.max(pad, info.pct.length);
-		}
+		for (const [ label, data ] of Object.entries(usage)) {
+			const metrics = Object.values(data);
+			if (!metrics.length) {
+				continue;
+			}
 
-		// print the usage
-		for (const { color, name, pct, quota, unit, used, value } of metrics) {
-			table.push([
-				name,
-				highlight(`${format(value)} / ${format(quota)} ${unit}`),
-				`${color('\u25A0'.repeat(used))}${gray('⠂'.repeat(width - used))} ${pct.padStart(pad)}%`
-			]);
-		}
+			table.push([ `\n${bold(label)}` ]);
 
-		console.log(table.toString());
+			// print the usage
+			for (const { envs, formatted, name, percent, unit, unlimited } of metrics) {
+				table.push([
+					`  ${bold(name)}`,
+					`${highlight(formatted.padStart(saasValuePadding))} ${unit}`,
+					unlimited || typeof percent !== 'number' ? '' : `${renderBar(percent, 20)}`
+				]);
+
+				// render the envs
+				for (let i = 0, len = envs.length; i < len; i++) {
+					const { formatted, name, production } = envs[i];
+					if (name !== 'default') {
+						table.push([
+							`  ${i + 1 === len ? '└─' : '├─'} ${name} ${production ? gray('Production') : ''}`,
+							`${highlight(formatted.padStart(saasValuePadding))} ${unit}`,
+							''
+						]);
+					}
+				}
+			}
+		}
+		if (table.length) {
+			console.log(table.toString());
```

Comment:
is a new line also needed here?

Reply:
Nope, we have the line break on line 161:
```js
table.push([ `\n${bold(label)}` ]);
```